pex/cli/commands/lock.py

```diff
@@ -584,7 +584,6 @@ def _update(self):
                 lock_file=attr.evolve(
                     lock_file,
                     pex_version=__version__,
-                    target_systems=SortedTuple(lock_updater.lock_configuration.target_systems),
```

Comment:
Originally I was gunning for `lock update --target-system ...` being able to amend the original target systems much like the recent change to make `lock update -p ...` accept adding new requirements and not just updating existing ones. So, you could start with a lock for Linux and Mac and years later add Windows support to the lock. That requires a bit of work though; so I deferred but had not fully cleaned up correctly.